mmdet3d/datasets/scannet_dataset.py

```diff
@@ -67,6 +72,17 @@ def __init__(self,
                  filter_empty_gt: bool = True,
                  test_mode: bool = False,
                  **kwargs):
+
+        # construct seg_label_mapping for semantic mask
+        seg_max_cat_id = self.METAINFO['seg_max_cat_id']
+        seg_valid_cat_ids = self.METAINFO['seg_valid_cat_ids']
+        neg_label = len(seg_valid_cat_ids)
+        seg_label_mapping = np.ones(
+            seg_max_cat_id + 1, dtype=np.int) * neg_label
+        for cls_idx, cat_id in enumerate(seg_valid_cat_ids):
+            seg_label_mapping[cat_id] = cls_idx
+        self.seg_label_mapping = seg_label_mapping
```

Comment:
The `label_mapping` obtained in `Seg3DDataset` is actually a dict, not array. We need to unify them